apps/core/app/[locale]/(default)/account/page.tsx

```diff
@@ -33,16 +36,32 @@ interface Props {
   params: {
     locale: LocaleType;
   };
+  searchParams: {
+    action?: 'reset_password';
+  };
 }
 
-export default async function AccountPage({ params: { locale } }: Props) {
+export default async function AccountPage({ params: { locale }, searchParams: { action } }: Props) {
   const session = await auth();
   const t = await getTranslations({ locale, namespace: 'Account.Home' });
+  const messages = await getMessages({ locale });
+  const Account = messages.Account ?? {};
 
   if (!session) {
     redirect('/login');
   }
 
+  if (action === 'reset_password') {
```

Comment:
Thoughts on having this as a different page? Should help simplify `change-password-form.ts`

Reply:
Do you mean to create separate page alongside `change-password-form` that will be used exclusively for logged-in Customers?🤔

Reply:
Thanks, guys. it's updated